org.mwc.cmap.plotViewer/src/org/mwc/cmap/plotViewer/editors/CorePlotEditor.java

```diff
@@ -532,6 +537,40 @@ public void propertyChange(final PropertyChangeEvent arg0) {
 		// };
 	}
 
+	protected String getFixedFilePath(final String fileName) {
+		String tiffFilePath = null;

```

Comment:
It would be better engineering to make this a `final String tiffFilePath` - then the compiler will help you in verifying that it gets assigned to a value (or null) in each logical branch.